validation-test/stdlib/Sort.swift.gyb

```diff
@@ -56,6 +56,12 @@ func expectSortedCollection(_ sortedAry: ArraySlice<Int>, _ originalAry: ArraySl
   expectSortedCollection([Int](sortedAry), [Int](originalAry))
 }
 
+func expectSortedCollection<C: Collection>(_ a: C,
+  by areInIncreasingOrder: (C.Element, C.Element) -> Bool
+) {
+  expectFalse(zip(a.dropFirst(), a).contains(where: areInIncreasingOrder))
```

Comment:
Of course you meant `a.lazy.dropFirst()` :-)

Reply:
I don't think so. It's just that I seem to have developed this habit of trying to apply `.lazy` everywhere and see if that'd work. Works in this case. Will perhaps make the test a millisecond faster.